src/plugin/memory-webview-main.ts

```diff
@@ -86,7 +87,34 @@ export class MemoryWebview {
         );
     };
 
-    public async show(initialMemory?: Partial<DebugProtocol.ReadMemoryArguments>): Promise<void> {
+    public openCustomDocument(uri: vscode.Uri): vscode.CustomDocument {
+        return {
+            uri,
+            dispose: () => {}
+        };
+    }
+
+    public async resolveCustomEditor(document: vscode.CustomDocument, webviewPanel: vscode.WebviewPanel): Promise<void> {
+        /*
+            memoryReference = debugprotocol.variable.memoryReference
+            displayName = 'memory'
+            DEBUG_MEMORY_SCHEME = 'vscode-debug-memory'
+            sessionId = <debug session ID>
+            range = undefined
+
+            document.uri is:
+
+            scheme: DEBUG_MEMORY_SCHEME,
+            authority: sessionId,
+            path: '/' + encodeURIComponent(memoryReference) + `/${encodeURIComponent(displayName)}.bin`,
```

Comment:
This piece seems to be causing some trouble for the CDT-GDB Adapter. Since its memory references are `&(variable)` rather than the address, this is turning it into `%26(variable)`. We may want to `decodeUriComponent` before we pass it into the Memory Inspector, or consider the API a bit.

Reply:
I've added decodeURIComponent